app/assets/stylesheets/all/option_sets.css.scss

```diff
@@ -109,28 +103,4 @@ form.option_set_form {
    (not inside the form since it's appended to the body) */
 
 div.modal.edit-named-item {
```

Comment:
I set this up, so styling should apply. But the edit option info needs to be placed inside an Elmo form to be styled with the styles. Right now, it looks pretty basic.